alchemist/alchemist-incarnation-biochemistry/src/main/java/it/unibo/alchemist/model/implementations/actions/ChemiotacticPolarization.java

```diff
@@ -0,0 +1,139 @@
+package it.unibo.alchemist.model.implementations.actions;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.math3.util.FastMath;
+
+import it.unibo.alchemist.model.implementations.molecules.Biomolecule;
+import it.unibo.alchemist.model.implementations.positions.Continuous2DEuclidean;
+import it.unibo.alchemist.model.interfaces.CellNode;
+import it.unibo.alchemist.model.interfaces.Context;
+import it.unibo.alchemist.model.interfaces.Environment;
+import it.unibo.alchemist.model.interfaces.EnvironmentNode;
+import it.unibo.alchemist.model.interfaces.Node;
+import it.unibo.alchemist.model.interfaces.Position;
+import it.unibo.alchemist.model.interfaces.Reaction;
+
+/**
+ *
+ */
+public class ChemiotacticPolarization extends AbstractAction<Double> {
```

Comment:
I believe this should be `ChemotacticPolarization` (without "i") in correct English. @smontagna, am I correct?